ktlint-rules/src/main/kotlin/software/aws/ktlint/rules/CustomRuleSetProvider.kt

```diff
@@ -1,13 +1,12 @@
 package software.aws.ktlint.rules
 
-import com.pinterest.ktlint.core.RuleSet
-import com.pinterest.ktlint.core.RuleSetProvider
+import com.pinterest.ktlint.core.RuleProvider
+import com.pinterest.ktlint.core.RuleSetProviderV2
 
-class CustomRuleSetProvider : RuleSetProvider {
-    override fun get() = RuleSet(
-        "custom-ktlint-rules",
-        CopyrightHeaderRule(),
-        ExpressionBodyRule(),
-        MultilineIfElseBlockRule(),
+class CustomRuleSetProvider : RuleSetProviderV2("custom-ktlint-rules", NO_ABOUT) {
+    override fun getRuleProviders() = setOf(
+        RuleProvider { CopyrightHeaderRule() },
+        RuleProvider { ExpressionBodyRule() },
+        RuleProvider { MultilineIfElseBlockRule()},
```

Comment:
**Style**: Missing space before closing `}`. Shouldn't **ktlint** itself have caught this? 🤔

Reply:
It seems like the ktlint rules are not being run on ktlint configuration files, I will see if there's an easy way to enable that